ArcPicker/ArcPicker.js

```diff
@@ -155,6 +169,23 @@ const ArcPickerBase = kind({
 		className: 'arcPicker'
 	},
 
+	handlers: {
+		onKeyDown: (ev, props) => {
+			const {children, disabled, onClick, value} = props;
+			const index = children.findIndex(child => child === value);
+
+			forward('onKeyDown', ev, props);
+
+			if (!disabled) {
+				if (isDown(ev.keyCode)) {
+					onClick(children[Math.max(index - 1, 0)])(ev);
```

Comment:
Should we use `onClick` here? how about `onChange` like `ArcSlider`?

Reply:
As we discussed, let's leave it as-is because `onClick` is defined as public now.